src/types.rs

```diff
@@ -883,3 +883,57 @@ where
     }
     deserializer.deserialize_any(HeightVisitor)
 }
+
+#[derive(Clone, Debug, PartialEq, Serialize, Default)]
+pub struct WalletCreateFundedPsbtOptions {
+    #[serde(rename = "fee_rate", skip_serializing_if = "Option::is_none")]
+    pub fee_rate: Option<f64>,
+    #[serde(rename = "lock_unspent", skip_serializing_if = "Option::is_none")]
+    pub lock_unspent: Option<bool>,
+    #[serde(rename = "conf_target", skip_serializing_if = "Option::is_none")]
+    pub conf_target: Option<u16>,
+    #[serde(rename = "replaceable", skip_serializing_if = "Option::is_none")]
+    pub replaceable: Option<bool>,
+}
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct WalletCreateFundedPsbt {
+    pub psbt: String,
+    #[serde(deserialize_with = "deserialize_bitcoin")]
+    pub fee: Amount,
+    #[serde(rename = "changepos")]
+    pub change_pos: i32,
+}
```

Comment:
Can we add `locktime` as well?
We don't have a `deserialize_locktime` but you are free to include, otherwise we can tackle in a follow-up PR. If you don't want to add it please open a ticket for this?
If you want to go ahead just note that this might be tricky because you'll be deserializing an unsigned int as either an `{relative,absolute}::LockTime` (see https://docs.rs/bitcoin/latest/bitcoin/absolute/enum.LockTime.html and https://docs.rs/bitcoin/latest/bitcoin/relative/enum.LockTime.html) and you'll need to have the proper safeguards (and tests!)